package.json

```diff
@@ -75,29 +75,29 @@
     "date-fns-tz": "^2.0.0",
     "deepmerge": "^4.3.1",
     "eventsource": "^2.0.2",
-    "fast-json-stringify": "^5.16.0",
-    "fastify": "^4.28.0",
+    "fast-json-stringify": "^6.0.0",
```

Comment:
This also didn't switch anything important?

Reply:
nope, just fastify update under the hood, we are on proper version